src/VssCommandSet.cpp

```diff
@@ -62,11 +62,15 @@ std::string VssCommandProcessor::processSet2(WsChannel &channel,
   //(set all or none)
   for ( std::tuple<VSSPath,jsoncons::json> setTuple : setPairs) {
     if (! database->pathExists(std::get<0>(setTuple) )) {
+      stringstream msg;
+      msg << "Path " << std::get<0>(setTuple).to_string() << " does not exist";
+      logger->Log(LogLevel::VERBOSE,msg.str());
```

Comment:
VERBOSE enough? INFO or ERROR or WARNING better?

Reply:
Good point. ERROR not. WARNING or INFO I think. Any opinions? (I woudl try to also make this consistent with get and aother erros then)

Reply:
I think WARNING is better. thx

Reply:
Changed. I tried to use "WARNING" for errors such as element not found or no access rights. I keep "ERROR" for syntactically invalid JSON and for  unexpected errors (i.e. in the catch std::exception clauses, where we actually do not expect to land)